backend/app/tests/api/routes/test_content.py

```
@@ -0,0 +1,118 @@
+        user_id=user_id_for_test  # Ensure user_id is included as per schema
+    )
+    
+    # Mock the response from the CRUD function
+    mock_created_item = create_mock_content_item_public(
+        item_id=item_id_for_test, 
+        user_id=user_id_for_test, 
+        title=content_item_create_data.title
+    )
+    
+    mocker.patch("app.api.routes.content.crud_create_content_item", return_value=mock_created_item)
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)